provisioner.go

```diff
@@ -180,8 +180,11 @@ func (p *LocalPathProvisioner) Provision(opts pvController.ProvisionOptions) (*v
 	if err != nil {
 		return nil, err
 	}
+
 	name := opts.PVName
-	path := filepath.Join(basePath, name)
+	folderName := strings.Join([]string{name, opts.PVC.Namespace, opts.PVC.Name}, "-")
```

Comment:
Use `_` instead of `-`? Make it easier to tell apart different parts of the name.